### DISCUSSION TITLE
Why doesn't this work for me?

### DISCUSSION THREAD

**Discussion by MelGrubb** [dotnet/roslyn#49465] [General]
I've been experimenting with the source generators for a week or so, and I'm just not seeing the same behavior that I see in the videos or read in blogs. I've tried using the current preview version of VS as well as the final release of 16.8.1, but I still have to restart constantly to get Intellisense to stop flagging everything as an error. I can generate code, and my usages compile, but they're all marked as errors in Intellisense. If I restart VS, everything is fine until the next compile when it all goes red again.

Is there some important detail that everyone's forgetting to mention?

**Comment by CyrusNajmabadi** [comment]
Source generators in VS are effectively pre-alpha.  Please try the latest 16.9 preview builds and report issues if you run into them.  Thanks!

**Reply by MelGrubb** [reply]
I am using the latest preview VS I can find, and it still doesn't work for me the way I see it in the demonstrations.
